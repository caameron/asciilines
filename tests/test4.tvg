3 3 3
* 1 1 h 3
