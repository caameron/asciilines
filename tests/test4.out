Invalid number of argument on first line of file for canvas size. Must be two numbers seperated by space
